src/main/webapp/index.html

```diff
@@ -377,15 +392,26 @@ <h3 class="col text-muted">WebRTC Publish</h3>
 						$("#dataMessagesTextarea").append("Received: " + obj.event.data + "\r\n");
 					}
 					else if (info == "available_devices") {
-						var htmlContent = "";
+						var videoHtmlContent = "";
 						obj.forEach(function(device) {
 							if (device.kind == "videoinput") {
-								htmlContent += getCameraRadioButton(device.label, device.deviceId);
+								videoHtmlContent += getCameraRadioButton(device.label, device.deviceId);
 							}
 						}); 
-						$(htmlContent).insertAfter(".video-source-legend");
+						$(videoHtmlContent).insertAfter(".video-source-legend");
 
 						$(".video-source").first().prop("checked", true);	
+						
+						var audioHtmlContent = "";
+						obj.forEach(function(device) {
```

Comment:
I think you can use the above `obj.forEach` method for this. No need to iterate separately for video and audio